src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -30,18 +30,6 @@ public class OSIsoftServer {
   /** Boolean to represent when the last sent message was succesfully sent. */
   private static boolean connected = true;
 
-  /** Ewon error code representing a send error */
-  public static final int SEND_ERROR = 32603;
-
-  /** Ewon error code representing a JSON error */
-  public static final int JSON_ERROR = -100;
-
-  /** Ewon error code representing a web ID error */
-  public static final int WEB_ID_ERROR = SCHttpUtility.HTTPX_CODE_AUTH_ERROR;
-
-  /** Ewon error code representing a generic error */
-  public static final int GENERIC_ERROR = 254;
-

```

Comment:
What is the value in removing these? could any of these be possibility needed in the future? In general im fine with leaving in unused constants that are ewon specific like this. Really I think its probably worth having an Ewon error constants file in the extension lib but thats a different can of worms.

Reply:
This was a code cleanup focused change, focused on decreasing errant code. I'm fine leaving these in if there is an argument for it, will drop this commit.